charts/consul/test/unit/client-config-configmap.bats

```diff
@@ -2,30 +2,23 @@
 
 load _helpers
 
-@test "client/ConfigMap: enabled by default" {
-  cd `chart_dir`
-  local actual=$(helm template \
-      -s templates/client-config-configmap.yaml  \
-      . | tee /dev/stderr |
-      yq 'length > 0' | tee /dev/stderr)
-  [ "${actual}" = "true" ]
-}
-
 @test "client/ConfigMap: enable with global.enabled false" {
   cd `chart_dir`
   local actual=$(helm template \
       -s templates/client-config-configmap.yaml  \
+      --set 'client.enabled=true' \
```

Comment:
Clients are now disabled by default (previously they were enabled), and so lots of bats tests needed to be changed to enable clients explicitly